src/icon/itineraryLocation.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import BaseIcon from '_utils/icon'
+import { BaseIconDefaultProps } from '_utils/icon/BaseIcon'
+
+export const ItineraryLocation = (props: Icon) => (
```

Comment:
@wakooka I can't see this icon in the design system icon assets list. Is it something new? 

Otherwise I found the name a bit confusing as we already have an itinerary component.

Reply:
Let's use the `Bullet` component

Reply:
I agree ItineraryLocation name is very confusing.
Anyway, I'll use `Bullet` component instead of it.

Reply:
Thanks @wakooka
`Bullet` is perfect for the case (`Circle` is too thin)
I've already called designer to confirm it